### PR TITLE
Release CLI Fixes

### PR DESCRIPTION
**Pull request by jshcrowthe**
Fixes some issues/deficiencies in the release CLI

1. Fixes an issue where `--follow-tags` wasn't functioning as expected
1. Added some flags to skip various parts of the build process

### CODE REVIEW

File: scripts/release/utils/git.js [hunk 1]
```diff
@@ -30,23 +30,40 @@ exports.cleanTree = async () => {
...
+ * Commits the current state of the repository and tags it with the appropriate
+ * version changes.
+ *
+ * Returns the tagged commits
+ */
 exports.commitAndTag = async (updatedVersions, releaseType) => {
-  await git.add('*/package.json');
-  await git.commit(
-    releaseType === 'Staging' ? 'Publish Prerelease' : 'Publish'
+  await exec('git add */package.json');
```
Comment (hiranya911) [inline scripts/release/utils/git.js]: Revise the input args to this function. It doesn't look like `releaseType` is used any more.

File: scripts/release/utils/git.js [hunk 2]
```diff
@@ -30,23 +30,40 @@ exports.cleanTree = async () => {
...
+  );
+  return tags;
 };
 
-exports.pushUpdatesToGithub = async () => {
-  await git.push('origin', 'master', {
-    '--follow-tags': null,
-    '--no-verify': null
-  });
+exports.pushUpdatesToGithub = async tags => {
```
Comment (hiranya911) [inline scripts/release/utils/git.js]: (tags)

File: scripts/release/utils/git.js (hunk 1 above)
Comment (jshcrowthe) [reply scripts/release/utils/git.js]: Done!!

File: scripts/release/utils/git.js (hunk 2 above)
Comment (jshcrowthe) [reply scripts/release/utils/git.js]: This is more of a style thing. As it is not specified by prettier I'm going to leave as is.

**Pull request merged**